src/component/utils/exploration/__tests__/__snapshots__/wrapBlockNodes.js.snap

```diff
@@ -0,0 +1,799 @@
+  >
+    <blockquote
+      data-block={true}
+      data-editor="editor"
+      data-offset-key="A-0-0"
+    >
+      <div
+        className="public__DraftStyleDefault__block"
+        data-offset-key="A-0-0"
+      >
+        <ol
+          className="public__DraftStyleDefault__ol"
+          data-offset-key="B-0-0"
+        >
+          <li
+            data-block={true}
+            data-editor="editor"
+            data-offset-key="B-0-0"
+          >
+            <div
+              className="public__DraftStyleDefault__block"
+              data-offset-key="B-0-0"
+            >
+              <span
+                data-offset-key="B-0-0"
+                style={Object {}}
+              >
+                <span
+                  data-text="true"
+                >
+                  fist list item
+                </span>
+              </span>
+            </div>
+          </li>
+        </ol>
+      </div>
+    </blockquote>
+    <div
+      className="public__DraftStyleDefault__block"
+      data-offset-key="B-0-0"
+    >
+      <span
+        data-offset-key="B-0-0"
+        style={Object {}}
+      >
+        <span
+          data-text="true"
+        >
+          fist list item
+        </span>
+      </span>
+    </div>
+  </ol>
+</div>
+`;
+
+exports[`renders block with children that have same wrapperElement 1`] = `
+<div>
+  <ol
+    className="public__DraftStyleDefault__ol"
+    data-offset-key="A-0-0"
+  >
+    <blockquote
+      data-block={true}
+      data-editor="editor"
+      data-offset-key="A-0-0"
+    >
+      <div
+        className="public__DraftStyleDefault__block"
+        data-offset-key="A-0-0"
+      >
+        <ol
+          className="public__DraftStyleDefault__ol"
+          data-offset-key="B-0-0"
+        >
+          <li
+            data-block={true}
+            data-editor="editor"
+            data-offset-key="B-0-0"
+          >
+            <div
+              className="public__DraftStyleDefault__block"
+              data-offset-key="B-0-0"
+            >
+              <span
+                data-offset-key="B-0-0"
+                style={Object {}}
+              >
+                <span
+                  data-text="true"
+                >
+                  fist list item
+                </span>
+              </span>
+            </div>
+          </li>
+          <li
+            data-block={true}
+            data-editor="editor"
+            data-offset-key="C-0-0"
+          >
+            <div
+              className="public__DraftStyleDefault__block"
+              data-offset-key="C-0-0"
+            >
+              <span
+                data-offset-key="C-0-0"
+                style={Object {}}
+              >
+                <span
+                  data-text="true"
+                >
+                  second list item
+                </span>
+              </span>
+            </div>
+          </li>
+        </ol>
+      </div>
+    </blockquote>
+    <div
```

Comment:
Similarly here - why is there another div with list items after the `blockquote`? And why is there an `ol` wrapping the `blockquote`?